book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Kindly note that it is a requirement that a method for adding rentals should be present in the book class. Please add it as well. 
e.g
```
def add_rental(person, date)
   
    Rental.new(date, self, person)
  end
```